app/src/main/java/org/mifos/mobile/ui/fragments/AccountsFragment.kt

```diff
@@ -207,6 +207,14 @@ class AccountsFragment : BaseFragment(), OnRefreshListener, AccountsView, Recycl
         currentFilterList = null
     }
 
+    /**
+     * This method is called when the fragment is visible to the user and actively running.
```

Comment:
Let's remove this javadoc comment. Its the same as the parent class method which we are overriding

Reply:
Thanks for pointing this out, I've made the changes :)